src/Frontend.hs

```diff
@@ -172,15 +172,15 @@ type AulaMain =
 
 aulaMain :: ActionM m => ServerT AulaMain m
 aulaMain =
-       runHandler Page.viewRooms
+       forceChangeInitialPassword (runHandler Page.viewRooms)
```

Comment:
My first idea was that. I wanted to code that into the `runHandler`. What would happen with that approach:

 1. Render login page
 1. User logs in
 1. runHandler on list ideas checks if there is an initial password, it redirects to the user setting page
 1. the server receives a GET request for user settings page
 1. runHandler on the user profile page checks if there is initial password, it redirects to the user settings page
 1. and so on